smart/forward_model/model_fit.py

```diff
@@ -87,7 +87,7 @@ def makeModel(teff, logg=5, metal=0, vsini=1, rv=0, tell_alpha=1.0, airmass=1.0,
 
     #elif data is not None and instrument == 'apogee':
     elif instrument == 'apogee':
-        model    = smart.Model(teff=teff, logg=logg, metal=metal, modelset=modelset, instrument=instrument)
+        model    = smart.Model(teff=teff, logg=logg, metal=metal, modelset=modelset, instrument=instrument, order='all')
```

Comment:
For some reason, this piece of code is not the same as the main branch. I have handling for Sonora models there as well

Reply:
This looks correct to me. I have all the Sonora model handling in the interpolate function I believe.

Reply:
I think my question is that is it the right version compared to the most recent version on model_fit.py?
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/22515968/223877383-29d2c9c1-e7bf-4303-9035-719bd6743595.png">

Reply:
Yes, because I have the InterpolateModel function set metallicity to 0 for Sonora models already (no need to do that step here).

Reply:
I see. Since this branch might not be able to merge due to inconsistent versions, I could change with just a few lines on the main branch, but another option is to make a new branch and fold these changes in.